### DISCUSSION TITLE
Automatic client configuration for iOS

### DISCUSSION THREAD

**Discussion by MisterS89** [SafeExamBrowser/seb-mac#240] [Q&A] [answered]
Hello,

I'm having problems with the automatic client configuration for iOS. The documentation seems pretty clear to me.

https://safeexambrowser.org/ios/ios_usermanual_en.html
https://safeexambrowser.org/developer/seb-automatic-client-configuration.html

I have placed the SebClientSettings.seb (for configuring client, generated on Windows) as a test in the following locations:

subdomain.mainsite.be/safeexambrowser/SebClientSettings.seb
mainsite.be/safeexambrowser/SebClientSettings.seb
seb.subdomain.mainsite.be/safeexambrowser/SebClientSettings.seb

But as soon as I use my maindomain.be or subdomain.maindomain.be in SEB for iOS I always get the message that no config can be found.

Anyone have concrete experience with this?

**Comment by danschlet** [comment, marked as answer]
I don't know if every web server uses case sensitive filename paths, but try to rename your Windows-generated client config file to the correct case sensitivity mentioned in the documentation: **SEBClientSettings.seb**

**Comment by MisterS89** [comment]
Hi danschlet,

How could I have overlooked that! Of course this is case sensitive on a Linux server. Works perfectly now!

Thank you!
